src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -82,24 +82,22 @@ public void Initialize()
             // no-op
         }
 
-        public void Prepare(CancellationToken token)
+        public void Prepare(IEnumerable<IBindingConfig> bindingConfigs, CancellationToken token)
```

Comment:
Simplifying this method: it doesn't need to be passed the data in a previous method. It can just be passed as a parameter in this one.

I'll remove the unnecessary methods later.